lib/pushmi_pullyu/cli.rb

```diff
@@ -59,13 +59,15 @@ def configure_rollbar
     Rollbar.configure do |config|
       config.enabled = false unless options[:rollbar][:token].present?
       config.access_token = options[:rollbar][:token]
-
+      config.exception_level_filters.merge!({
```

Comment:
Add IOError as to be ignored so this particular error message will not be sent to Rollbar.